app/javascript/pages/HomePage.tsx

```diff
@@ -11,13 +11,14 @@ import {
 } from "@bloom-housing/ui-components"
 import Head from "next/head"
 
+import { withAppSetup } from "../layouts/app_settings"
 import Layout from "../layouts/Layout"
 
 export interface HomePageProps {
   listings?: Listing[]
 }
 
-const HomePage = (props: HomePageProps) => {
+const component = (props: HomePageProps) => {
```

Comment:
I think we want to keep this named `HomePage`. Question about the `export { HomePage as default, HomePage }` pattern: Why not just prefer `export default HomePage`? At my last job we would prefer a default export when a file only exported one thing, but I could just be missing why it's helpful to also provide a named export

Reply:
That is my preference as well, but how would you use `withAppSetup` in that case without causing an error and keeping with our linting rules? Cause I couldn't make that happen.

Reply:
@software-project let me try it out now. At my last job we used this pattern a lot and would just do

```
export default withAppSetup(HomePage)
```

but maybe that doesn't work with our linting rules

Reply:
Yeah I'm not getting any linting errors when I do:

```
const HomePage = (props: HomePageProps) => {
... component rendering ...
}

export default withAppSetup(HomePage)
```

Reply:
Ok, weird. Maybe there was a bug with my vs code linting, I'm like 90% sure I tried that.